.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Before opening an issue**
+Questions should go to the GEE mailing list (https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/gee-oss)  or Slack Open GEE channel (http://slack.opengee.org/)
```

Comment:
When Rendered this and the Line above run together. Add a Space between them.

Reply:
Done